components/org.wso2.carbon.identity.cors.service/src/main/java/org/wso2/carbon/identity/cors/service/internal/impl/CORSManagerImpl.java

```diff
@@ -121,6 +137,34 @@ public CORSConfiguration getCORSConfiguration(String tenantDomain) throws CORSMa
         }
     }
 
+    /**
+     * Select highest priority CORSOrigin DAO from an already sorted list of CORSOrigin DAOs.
+     *
+     * @return Highest priority CORSOrigin DAO.
+     */
+    private CORSOriginDAO getCORSOriginDAO() throws CORSManagementServiceException {
+
+        if (!this.corsOriginDAOS.isEmpty()) {
+            return corsOriginDAOS.get(corsOriginDAOS.size() - 1);
```

Comment:
So the highest number denoting the higher priority? Is the `corsOriginDAOS` list a sorted list based on the priority? Can you add a comment explaining how the priority order get resolved?

Reply:
Removed this feature.